extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryAdapter.java

```diff
@@ -100,11 +103,56 @@ public static Builder repositoryAdapter() {
       final RepositoryPresenter<Object> untypedPresenter =
           (RepositoryPresenter<Object>) checkNotNull(presenter);
       presenters.add(untypedPresenter);
+      isDynamicPresenter.add(true);
       observables.add(untypedRepository);
       return this;
     }
 
     /**
+     * Specifies that the {@link RepositoryAdapter} being built should present the given
+     * {@code item} next (after all previously added item), using the given
+     * {@code presenter} for any presentation logic. Added items will be considered static;
+     * any stable id added returned by the {@link RepositoryPresenter} will be ignored, and the
+     * item will be kept stable by the {@link RepositoryAdapter}
+     *
+     * @param item A static item. This can be the same as a previously added item; this makes the
+     * resulting {@link RepositoryAdapter} present the same data in different positions and/or
+     * different ways.
+     * @param presenter The repository presenter associated with the {@code repository} at this
+     *     position.
+     * @return This instance, for chaining.
+     */
+    @NonNull
+    public <T> Builder addItem(@NonNull final T item,
+        @NonNull final RepositoryPresenter<T> presenter) {
+      @SuppressWarnings("unchecked")
+      final Repository<Object> repository = (Repository<Object>) repository(item);
+      repositories.add(repository);
+      @SuppressWarnings("unchecked")
+      final RepositoryPresenter<Object> untypedPresenter =
+          (RepositoryPresenter<Object>) checkNotNull(presenter);
+      presenters.add(untypedPresenter);
+      isDynamicPresenter.add(false);
+      return this;
+    }
+
+    /**
+     * Specifies that the {@link RepositoryAdapter} being built should present the given
+     * {@code presenter} next (after all previously added items).
```

Comment:
either I use the long mapping as in the prev. commit, or I can revert it and do the essay here instead. up to you :)